composer.json

```diff
@@ -117,7 +117,7 @@
             "phpunit"
         ],
         "verify-callmap": "phpunit tests/Internal/Codebase/InternalCallMapHandlerTest.php",
-        "psalm": "@php ./psalm --find-dead-code",
+        "psalm": "@php ./psalm --no-cache -m",
```

Comment:
what was the reason for this change?

Reply:
`--find-dead-code` is on already in the config

`--no-cache` I was getting some weird errors appears, cache probably got into some weird state

`-m` phpstorm's run composer script terminal does not support ansi :(

I will revert the two additions